### DISCUSSION TITLE
Unable to Search Resources in Global Search

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#711] [General Discussion]
Hello Team,

We are currently using release 5.1 and have encountered an issue with the global search functionality. While attempting to search for resources, we're facing difficulties. Specifically, we can successfully search for courses, but when attempting to search for resources, the search yields no results.
I have also tried with add "ALL" tab in menu bar. 

Expected Behavior:
When using the global search feature, both courses and resources should be searchable, providing relevant results for both.

Actual Behavior:
While courses are successfully searchable, resources are not yielding any search results.

Screenshot:
Attached is a screenshot demonstrating the issue encountered during the search process.

This screenshort is for the course search
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/a0d70781-07d0-4f7a-a39e-bd6024b1fd0a)

This screenshort is for the resource search
![image](https://github.com/Sunbird-Ed/Community/assets/72926048/9421cff2-1abf-4431-a17f-9ef21c726ef1)

We appreciate any assistance in resolving this issue promptly. If further information is required, please let us know.

@vinukumar-vs , @rajeevsathish, @AmiableAnil , @Krishnaj20, @chitranshu-keshav, @SanthoshVasabhaktula, @coolbung, @komalm @paritshivani @manishakapadnis

**Comment by komalm** [comment]
@siddhishinde0723 

### Include All Content Types:
Modify the primary category to include all content types, not just **courses**. This ensures that when users search, they can find content from various content types.

### Update Menu API for "All" Menu:
**Updated Primary Category:**
In the menu API for the "All" menu, ensure that the updated primary category is with all content types. 

```
primaryCategory: [
    "Collection",
    "Resource",
    "Content Playlist",
    "Course",
    "Course Assessment",
    "Digital Textbook",
    "eTextbook",
    "Explanation Content",
    "Learning Resource",
    "Lesson Plan Unit",
    "Practice Question Set",
    "Teacher Resource",
    "Textbook Unit",
    "LessonPlan",
    "FocusSpot",
    "Learning Outcome Definition",
    "Curiosity Questions",
    "MarkingSchemeRubric",
    "ExplanationResource",
    "ExperientialResource",
    "Practice Resource",
    "TVLesson",
    "Course Unit",
    "Exam Question"
]
```

**Reply by siddhishinde0723** [reply]
Thanks @komalm the issue is solved.
